README.md

```diff
@@ -218,13 +218,13 @@ If you want to explicitly view the running log for a published integration in a
 
 ![Apache Camel K Integrations view - Follow log](images/camelk-integrations-view-remove-menu.jpg)
 
-A new Output channel opens. It is named for the running Kubernetes pod where the integration is running. This log updates as new data is added:
+A new view opens. It is named for the running Kubernetes pod where the integration is running. This log updates as new data is added:
 
-![Apache Camel K Integrations view - Integration log](images/camelk-integrations-view-integrations-log.jpg)
+![Apache Camel K Integrations view - Integration log](images/camelk-integrations-view-integrations-log-revised.jpg)
 
-## Stopping an Apache Camel K integration
+Note: By default, `Auto-scroll` is checked and new entries in the log will automatically be scrolled to at the bottom of the window. If turned off, the scrolling stops and can be done manually.
```

Comment:
I disagree. The sentence makes sense as is. "new entries in the log will automatically be scrolled to at the bottom of the window". Leaving it.

Reply:
You are the native speaker. Fine for me.

Reply:
fine for me too. you're the native speaker.
I reported because it feels strange to have two prepositions and Google doc autocorrection is suggesting to remove the 'at'